include/deal.II/base/config.h.in

```diff
@@ -39,6 +39,7 @@
 #cmakedefine DEAL_II_WITH_ARPACK
 #cmakedefine DEAL_II_WITH_ARBORX
 #cmakedefine DEAL_II_WITH_ASSIMP
+#cmakedefine FEATURE_BOOST_BUNDLED_CONFIGURED
```

Comment:
I wonder what @tamiko has to say about this. The name is likely obscure enough that it doesn't matter, but it's not namespaced...